src/components/share-structure/EditShareStructure.vue

```diff
@@ -240,11 +240,11 @@ export default class EditShareStructure extends Mixins(CurrencyLookupMixin) {
         (v: string) => !(this.shareClasses
           .find((s, index) => {
             // Don't apply uniqueness check to self
-            return index !== this.activeIndex && s.name.split(' Shares')[0].toLowerCase() === v.toLowerCase()
+            return index !== this.activeIndex && s.name?.split(' Shares')[0].toLowerCase() === v.toLowerCase()
```

Comment:
What problem is this solving?

(I'm wary of conditional chaining because it can hide things.)

Reply:
It's solving the unit test failing, there's an error regarding calling split on an undefined object

Reply:
reverting this back

Reply:
Yeah, better to fix the unit test than compromise the regular build.

Reply:
Probably `s` exists but is missing the `name` property in the unit test.

Reply:
I removed, wasn't a problem, problem was the v's below